### PR TITLE
Change well designs and add horizontal definition item

### PR DESCRIPTION
**Pull request by Tankenstein**
Horizontal definition item

image

Well

image

### CODE REVIEW

File: docs/_includes/components/badges.html [hunk 1]
```diff
@@ -16,15 +16,6 @@ <h1 id="badges" class="page-header">Badges</h1>
 <button class="btn btn-primary" type="button">
   Messages <span class="badge">4</span>
 </button>
-{% endhighlight %}
```
Comment (oliverviljamaa) [inline docs/_includes/components/badges.html]: Why?

File: docs/_includes/components/wells.html [hunk 2]
```diff
@@ -2,32 +2,32 @@
   <h1 id="wells" class="page-header">Wells</h1>
 
   <h2>Default well</h2>
-  <p>Use the well as a simple effect on an element to give it an inset effect.</p>
+  <p>Use the well as a simple effect on an element to make it a little informative card.</p>
   <div class="bs-example" data-example-id="default-well">
-    <div class="well">
+    <div class="well-xs">
```
Comment (oliverviljamaa) [inline docs/_includes/components/wells.html:7]: Don't we want to encourage `well well-xs` instead?

File: docs/_includes/components/wells.html [hunk 3]
```diff
@@ -2,32 +2,32 @@
   <h1 id="wells" class="page-header">Wells</h1>
 
   <h2>Default well</h2>
-  <p>Use the well as a simple effect on an element to give it an inset effect.</p>
+  <p>Use the well as a simple effect on an element to make it a little informative card.</p>
```
Comment (oliverviljamaa) [inline docs/_includes/components/wells.html:5]: Would this be a good time to backport `card` from Bootstrap 4?

File: docs/_includes/customizer-variables.html [hunk 4]
```diff
@@ -2689,7 +2689,7 @@ <h2 id="badges">Badges</h2>
   </div>
   <div class="col-xs-4">
     <label for="input-@badge-border-radius">@badge-border-radius</label>
-    <input id="input-@badge-border-radius" type="text" value="10px" data-var="@badge-border-radius" class="form-control"/>
+    <input id="input-@badge-border-radius" type="text" value="3px" data-var="@badge-border-radius" class="form-control"/>
```
Comment (oliverviljamaa) [inline docs/_includes/customizer-variables.html:2692]: Why do we change badge border radius?

File: docs/dist/js/bootstrap.min.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
 /*!
- * Bootstrap v5.13.1 (http://bootstrap.transferwise.com)
+ * Bootstrap v5.13.0 (http://bootstrap.transferwise.com)
```
Comment (oliverviljamaa) [inline docs/dist/js/bootstrap.min.js:2]: Interesting that the version is changing from `5.13.1` to `5.13.0` here?

File: docs/_includes/components/wells.html (hunk 2 above)
Comment (Tankenstein) [reply docs/_includes/components/wells.html:7]: No, that's styling, this is responsive. Thing about col behaviour, or text-center behaviour

File: docs/dist/js/bootstrap.min.js (hunk 5 above)
Comment (OFFLlNE) [reply docs/dist/js/bootstrap.min.js:2]: Yea, I just had 1 file where it went from 5.13.6 to 5.13.1  (where 5.13.1) was the latest 😅

File: docs/_includes/customizer-variables.html (hunk 4 above)
Comment (Tankenstein) [reply docs/_includes/customizer-variables.html:2692]: I ran some build-y stuff? Guessing it was outdated

File: docs/dist/js/bootstrap.min.js (hunk 5 above)
Comment (Tankenstein) [reply docs/dist/js/bootstrap.min.js:2]: Ugh

File: docs/_includes/components/badges.html (hunk 1 above)
Comment (Tankenstein) [reply docs/_includes/components/badges.html]: No idea lol, will put it back

**Comment by Tankenstein** [comment]
Closing as too lazy to merge. New pr at: https://github.com/transferwise/bootstrap/pull/153

**Pull request closed without merging**